lib/tasks/organizations.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+namespace :organizations do
+  namespace :backfill do
+    desc 'Backfills partner mailer for organizations with default content'
+    task default_email_text: :environment do |task|
+      default_email_text = "<div class=\"trix-content\">\n  <div>%{partner_name},<br><br>Your diaper request has been approved and you can find attached to this email a copy of the distribution you will be receiving.<br><br>Your distribution has been set to be <strong>%{delivery_method}</strong> on <strong>%{distribution_date}</strong>.<br><br>See you soon!<br><br>%{comment}</div>\n</div>\n"
```

Comment:
One small change please! Can you change it from `diaper` to `essentials`? I know the original mail had diaper but we've been trying to change the language!

Then we can merge!